src/app/services/serviceLocator.js

```diff
@@ -15,6 +15,14 @@ class ServiceLocator {
     return this._resolveService('_userService', './users/UserService', override)
   }
 
+  /**
+   * @param {Object} override
+   * @returns {AuthService}
+   */
+  authService(override = null) {
```

Comment:
This class helps with Dependency Injection, which doesn't have any great solutions in NodeJs (unless you use Typescript, which I was considering messing with but wanted to avoid another rabbit hole)